Review and add tips from notion doc (#1053)
wandb/docs: content/guides/models/track/runs/_index.md

For projects that contain a large number of runs, you can use either the search bar to filter runs you want to delete using Regex or the filter button to filter runs based on their status, tags, or other properties. 
{{% /alert %}}

## Organize runs 

> This section provides instructions on how to organize runs using groups and job types. By assigning runs to groups (e.g., experiment names) and specifying job types (e.g., preprocessing, training, evaluation, debugging), you can streamline your workflow and improve model comparison.

Author: Addressed in https://github.com/wandb/docs/pull/1053/commits/764f435f888a06354837d2869524e0c3d7a5a053

For projects that contain a large number of runs, you can use either the search bar to filter runs you want to delete using Regex or the filter button to filter runs based on their status, tags, or other properties. 
{{% /alert %}}

## Organize runs 

This section provides instructions on how to organize runs using groups and job types. By assigning runs to groups (for example, experiment names) and specifying job types (for example, preprocessing, training, evaluation, debugging), you can streamline your workflow and improve model comparison.

> ### Assigning runs to groups and job types

Reviewer: ```suggestion
### Assign a group or job type to a run
```

Each run in W&B can be categorized by **group** and a **job type**:

> - **Group**: Represents a broader experiment category, making it easier to organize and filter runs.
> - **Job type**: Describes the function of the run, such as preprocessing, training, or evaluation.
> 

Reviewer: ```suggestion
- **Group**: a broad category for the experiment, used to organize and filter runs.
- **Job type**: the function of the run, such as `preprocessing`, `training`, or `evaluation`.

```

> In the following [example workspace](https://wandb.ai/stacey/model_iterz?workspace=user-stacey), a baseline model is trained using increasing amounts of data from the Fashion-MNIST dataset. The color coding in the workspace represents the amount of data used:

Reviewer: ```suggestion
The proceeding [example workspace](https://wandb.ai/stacey/model_iterz?workspace=user-stacey), trains a baseline model using increasing amounts of data from the Fashion-MNIST dataset. The workspace uses colorts to represent the amount of data used:
```
Maybe link to the dataset

> - **Yellow to dark green**: Increasing amounts of data for the baseline model.

Reviewer: ```suggestion
- **Yellow to dark green** indicate increasing amounts of data for the baseline model.
```

> - **Light blue to violet to magenta**: Increasing amounts of data for a more complex "double" model with additional parameters.

Reviewer: ```suggestion
- **Light blue to violet to magenta** indicate amounts of data for a more complex "double" model with additional parameters.
```

> Using W&B's filtering options and search bar, you can easily compare runs based on specific conditions, such as:

Reviewer: ```suggestion
Use W&B's filtering options and search bar to compare runs based on specific conditions, such as:
```

- Training on the same dataset.
- Evaluating on the same test set.

> Applying filters dynamically updates the **Table** view, allowing you to quickly identify performance differences between models. For example, you can determine which classes are significantly more challenging for one model compared to another.

Reviewer: ```suggestion
When you apply filters, the **Table** view is updated automatically. This allows you to identify performance differences between models, such as determining which classes are significantly more challenging for one model compared to another.
```
